test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```

Comment:
In the first project, it is not need to create test unit!
So remove all files in ```test``` folders then add below line to ```.gitignore```

```
test/*
```